.github/workflows/master.yaml

```diff
@@ -22,4 +22,4 @@ jobs:
       - name: Sanity Check
         run: |
           go vet ./...
-          go test ./...
+          go test -short ./...
```

Comment:
Is this intentionally disabled here?

Reply:
Yes, the integration tests won't pass as they do not have credentials configured for a system to test against.